roboteam_ai/src/control/positionControllers/ForcePosControl.cpp

```diff
@@ -63,7 +63,6 @@ PosVelAngle ForcePosControl::calculateForcePosVelAngle(const PosController::Robo
     PosVelAngle target;
     auto force = calculateForces(robot, targetPos, forceRadius);
     target.pos = targetPos;
-    target.vel = control::ControlUtils::velocityLimiter(force);
     return controlWithPID(robot, target);
```

Comment:
You still have to put `force` back in `target.vel`. Now you broke ForceposControl

Reply:
@RobZelluf What did you do here exactly? I don't completely understand the rationale behind removing this

Reply:
@RobZelluf

Reply:
@RobZelluf

Reply:
Looks fine

Reply:
@RobZelluf

Reply:
@RobZelluf